cmr-exchange/service-bridge/Dockerfile

```diff
@@ -1,2 +1,9 @@
-FROM openjdk:8u162-jre-slim
-MAINTAINER <http://cmr.earthdata.nasa.gov/>
+FROM clojure
```

Comment:
We've switched to using:
FROM clojure:openjdk-8-lein

so that we don't have unexpected versions of Java.